adflow/pyADflow.py

```diff
@@ -1897,8 +1888,8 @@ def solveSep(self, aeroProblem, sepStar, nIter=10, alpha0=None,
         # Name of function to use
         funcName = '%s_%s'%(ap.name, sepName)
 
-        if not self.getOption('rkreset') and self.getOption('usenksolve'):
-            ADFLOWWarning("nRKReset option is not set. It is usually necessary "
+        if not self.getOption('rkreset') and self.getOption('usenksolver'):
+            ADFLOWWarning("RKReset option is not set. It is usually necessary "
```

Comment:
Is this still true with ANK?

Reply:
I think so. RK is still the best way to increase the residual after restart.

Reply:
`'infchangecorrection':True` is also a great option (better than RK reset imo). I think we are planning to make that enabled by default in #41